Makefile

```diff
@@ -12,32 +12,31 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
-DOCKER=1
-
 .PHONY: build build-controller build-importer \
 		docker docker-controller docker-cloner docker-importer \
 		test test-functional test-unit \
 		publish \
 		vet \
 		format \
 		manifests \
-		goveralls
-
-all: docker
+		goveralls \
+		release-description
 
-clean:
+DOCKER=1
 ifeq (${DOCKER}, 1)
-	./hack/build/in-docker "./hack/build/build-go.sh clean; rm -rf bin/* _out/* manifests/generated/* .coverprofile"
+DO=./hack/build/in-docker
 else
-	./hack/build/build-go.sh clean; rm -rf bin/* _out/* manifests/generated/* .coverprofile
+DO=eval
```

Comment:
do you **need** `eval` for this case? IOW, would DO="#" work?

Reply:
I tried that initially, but without `eval` it's equivalent to `$ "./do/my/script.sh"` which isn't legal.